src/sage/rings/polynomial/infinite_polynomial_element.py

```diff
@@ -560,6 +560,44 @@ def is_nilpotent(self):
         """
         return self._p.is_nilpotent()
 
+    def numerator(self):
+        r"""
+        Return a numerator of ``self``, computed as ``self * self.denominator()``.
+
+        Note that some subclasses may implement its own numerator
+        function.
```

Comment:
```suggestion
        Note that some subclasses may implement their own numerator methods.
```

Reply:
Actually, this sentence should be removed.

I copied the description from `MPolynomial`.

Reply:
done below.